### PR TITLE
fix: remove invalid hasDotWunderGraphDirectory option

### PR DESCRIPTION
**Pull request by StarpTech**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by jensneuse** [comment]
So, this code is correct and we could merge it.
However, I think that the real problem is that we don't know in which directory the user installed `.wundergraph`.
Is the WunderGraph dir in the root of a project, or are we in a monorepo and have some nesting?

For VSCode it's crucial that the `.graphqlconfig` sits at the root of the project. Otherwise, IntelliSense won't work.
Ideally, we could configure an absolute or relative path where the `.graphqlconfig` should be generated and updated.

File: docs-website/src/pages/docs/wundergraph-config-ts-reference/configure-graphqlconfig.md [hunk 1]
```diff
@@ -4,27 +4,23 @@ pageTitle: 'WunderGraph - Configure .graphqlconfig'
 description:
 ---
 
-This section describes how to set up the configuration for generating the `.graphqlconfig` file.
-
-This file is very important for WunderGraph to work.
+This `.graphqlconfig` file is very important for WunderGraph to work.
```
Comment (jensneuse) [inline docs-website/src/pages/docs/wundergraph-config-ts-reference/configure-graphqlconfig.md:7]: Updated proposal:

This `.graphqlconfig` file is very important for an ideal developer experience.
IDEs like VSCode and Jetbrains recognise the file and use it to configure IntelliSense.
It's important to place it in the right directory.
E.g. in VSCode, it's important that you place it at the root of the project.
With the `.graphqlconfig` properly configured, your IDE helps you to define GraphQL Operations.

**Review by jensneuse — CHANGES_REQUESTED** [review]

**Comment by StarpTech** [comment]
> Ideally, we could configure an absolute or relative path where the .graphqlconfig should be generated and updated.

I tried this already, but it's not that easy to do because you user can only specify a relative path.

Example:

```ts
configureWunderGraphApplication({
  dotGraphQLConfig: {
    disable: true,
	path: '../../.graphqlconfig'
  },
})
```

How should we calculate the schema path? We would need to calculate the target directory and search for the nearest `.wundergraph` directory.

**Comment by jensneuse** [comment]
@StarpTech update from main+merge or discard?

**Comment by bmcilw1** [comment]
> So, this code is correct and we could merge it. However, I think that the real problem is that we don't know in which directory the user installed `.wundergraph`. Is the WunderGraph dir in the root of a project, or are we in a monorepo and have some nesting?
> 
> For VSCode it's crucial that the `.graphqlconfig` sits at the root of the project. Otherwise, IntelliSense won't work. Ideally, we could configure an absolute or relative path where the `.graphqlconfig` should be generated and updated.

Can both of these challenges be addressed by setting an ENV var on the startup of the CLI? I believe both VSCode plugins and the running CLI process could reference an ENV var if set by something similar to #300

**Comment by jensneuse** [comment]
@fiam what should we do with this? I think you were working on a similar issue.

**Comment by fiam** [comment]
@jensneuse I think this one can be closed now, superseded by #537

**Comment by StarpTech** [comment]
Yeah, let's close it.

**Pull request closed without merging**